packages/code-du-travail-frontend/next.config.js

```diff
@@ -10,10 +8,25 @@ const withTranspileModule = require("next-transpile-modules")([
   "p-debounce",
   "is-plain-obj",
 ]);
+
 const withBundleAnalyzer = require("@next/bundle-analyzer")({
   enabled: process.env.ANALYZE === "true",
 });
 
+const ContentSecurityPolicy = `
+default-src 'self' *.travail.gouv.fr *.data.gouv.fr *.fabrique.social.gouv.fr;
+img-src 'self' data: *.fabrique.social.gouv.fr https://travail-emploi.gouv.fr https://mon-entreprise.urssaf.fr ${
+  process.env.AZURE_BASE_URL
```

Comment:
cette var est undefined :
voila le header qu'on a :
`img-src 'self' data: *.fabrique.social.gouv.fr https://travail-emploi.gouv.fr https://mon-entreprise.urssaf.fr undefined;`

Reply:
Ahhh en effet, quand c'est non defini on met undefined, il faudrait faire 

```suggestion
img-src 'self' data: *.fabrique.social.gouv.fr https://travail-emploi.gouv.fr/ https://mon-entreprise.urssaf.fr/ ${
  process.env.AZURE_BASE_URL ?? ""}
```